Source/WebCore/Modules/WebGPU/GPUComputePassEncoder.cpp

```diff
@@ -48,12 +48,14 @@ void GPUComputePassEncoder::setPipeline(const GPUComputePipeline& computePipelin
     m_backing->setPipeline(computePipeline.backing());
```

Comment:
Changeling says "Explanation of why this fixes the bug (OOPS!)."

Reply:
I'll fix that, it should be replaced with `Reviewed by NOBODY (OOPS!).`